### DISCUSSION TITLE
Should we improve suppoert for sections not using the location taxonomy when using the Related content block - country tag thumbnail

### DISCUSSION THREAD

**Discussion by SophieAmnesty** [amnestywebsite/humanity-theme#245] [Enhancement Requests]
This has come up with the Slovenia site. 

When a site enables the related content block on posts. The first item in the related content block will be a country, if a country tag is being used. 

e.g. https://secsloveniastg.wpengine.com/samo-ja-pomeni-ja/ 
![image](https://github.com/amnestywebsite/humanity-theme/assets/99269799/f5d7f3bc-f3cf-4110-94dd-b083e83b0b68)
 

This links to a pre-filtered search results page for that country tag (unlike .org which has the country pages). e.g. https://secsloveniastg.wpengine.com/iskalnik/?qlocation=1620  

Suggestions from the Slovenia team are: 

1. enable them to add a thumbnail image for the country thumbnail in the related content block 
2. move the country thumbnail to the end of the list 
3. remove the country taxonomy from the related content block so it just shows other related posts and no country thumbnail

this will apply on any humanity theme sites using the related content block and the countries taxonomy. 
e.g. eurasia site: https://eurasia.amnesty.org/2024/06/14/opravdatelnyj-prigovor-po-kempir-abadskomu-delu-pobeda-spravedlivosti-i-prav-cheloveka-v-kyrgyzstane-zayavila-amnesty-international/ 
![image](https://github.com/amnestywebsite/humanity-theme/assets/99269799/57a65f1f-a368-4175-bfab-b0a7af2853c5)

**Comment by SophieAmnesty** [comment]
Slovenia colleagues asked about this again. 

I tried adding an image to the country term: 

![image](https://github.com/user-attachments/assets/8c685522-1ef3-4823-ab4c-149cb0100b9e)
 
But this didn't seem to fix the issue: 

![image](https://github.com/user-attachments/assets/3353a8d4-0210-4806-8246-f8d54113682d)
 
@AmnestyAM do you think we can fix this for them? Should I add this as a ticket instead of a discussion?

**Reply by AmnestyAM** [reply]
I will make sure to have a chat with @jaymcp about this tomorrow :-)

**Reply by jaymcp** [reply]
Since the Related Content block is already set up for supporting country pages, and the taxonomy exists within the theme, we should port over the featured image field for the term into the theme. This will allow users to attached featured images to location terms without us having to rewrite the Related Content renderer significantly.

**Reply by SophieAmnesty** [reply]
sounds perfect

**Comment by AmnestyAM** [comment]
added a ticket for this here -> https://github.com/amnestywebsite/humanity-theme/issues/324

**Comment by AmnestyAM** [comment]
this works, tested here https://iseurasiastg.wpengine.com/2024/11/14/test-2/ each country will need a featured image adding then we might need to clear the cache for each post. Is there a way I can do that in bulk @jaymcp ?

**Reply by jaymcp** [reply]
the easiest way to bulk-clear a cache is via the WPE dashboard for the installation. it should be sufficient to clear object & page caches
